students/drovdahl/codebat/warmup-1/parrot_trouble.py

```diff
@@ -0,0 +1,9 @@
+#! /usr/local/bin/python3
+
+
+def parrot_trouble(talking, hour):
+    if hour < 7 or hour > 20:
+        hour = True
+    else:
+        hour = False
+    return ((talking and hour))
```

Comment:
I wouldn't get in the habit of using hour to refer to an int at first and a bool later. Could be confusing!